views/drones/list.hbs

```diff
@@ -1 +1,19 @@
-<h2>Available drones</h2>
\ No newline at end of file
+<article>
+  <h2>Available drones</h2><a href="/drones/create">Create a new Drone</a>
+  <ul>
+    {{#each droneObj}}
+    <h3>Drone: {{this.name}}</h3>
+    <p>Propellers: {{this.propellers}}</p>
+    <p>Max. speed: {{this.maxSpeed}}</p>
+    <a href="/drones/{{this.id}}/edit">Edit</a>
+    
+    <a href="/drones/{{this.id}}/delete">
+    <button>DELETE</button></a>
```

Comment:
To make your delete work, you should wrap this in a form and place the button inside. It will work with the route handler you in your `drones.js`. Something like this

```suggestion
  <form action="/drones/{{this.id}}" method="post"> 
    <button>Delete</button>
  </form>
```